<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalInformation</class>
 <widget class="QWidget" name="PersonalInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)
</string>
   </property>
   <property name="text">
    <string>登录名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>医生姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>职称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="workLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>科室：</string>
   </property>
  </widget>
  <widget class="QLabel" name="placeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>坐诊时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>从业经验：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>77</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/img/doctorPt.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="expLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;新宋体&quot;;color:rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background:url(:/img/personalReturn.png);*/
border-style:hidden</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(100,123,34,90) ;
font: 75 9pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>信息介绍：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
